docs/user-guides/manual-setup.md

```diff
@@ -0,0 +1,162 @@
+# Partner chains setup
+
+Partner Chains provides text based CLIs, for setting up a partner chain and starting nodes, called wizards.
+In the [chain builder](./chain-builder.md), [permissioned node owner](./permissioned.md), and [registered node owner](./registered.md) guides these wizards are used.
+
+This document explains what is performed by those wizards and how to perform such operations manually.
+
+For dependency setup please see the respective guide for your desired setup.
+
+## Generating keys
+
+`<node-executable> wizards generate-keys` creates 3 specific keys, stores them in the keystore and also creates a JSON file with public keys.
+These 3 keys are:
+* Aura, sr25519 scheme key, with "key-type" `aura`
+* GRANDPA, ed25519 scheme key, with "key-type" `gran`
+* cross-chain, ecdsa scheme key, with "key-type" `crch`
+
+For each of these key commands that are two commands invoked by wizard.
+The first one is
+```bash
+<node-executable> key generate --scheme <scheme> --output-type json
+```
+, that has `secretPhrase` and `publicKey` fields in the output.
+Please do save these outputs as they are needed in the subsequent steps.
+
+The second one is
+```bash
+<node-executable> key insert --keystore-path <base-path>/keystore --scheme <scheme> --key-type <key-type> --suri <secretPhrase>
+```
+that stores the generated key in the keystore.
+
+Commands above serve as instructions for generating and inserting required key in the keystore.
+If there is a need for keys of different schema or type, or in different location, the instructions above should be used to insert it.
+
+The wizard can also generate node network key, using following commands
+```bash
+mkdir -p <base-path>/network
+mkdir key generate-node-key --file <base-path>/network/secret_ed25519
+```
+
+Please note that running node will require using `--base-path`, `--keystore-path` and `--node-key-file` to match the layout used above.
+
+## Prepare configuration
+
+`<node-executable> wizards prepare-configuration` sets up partner chain governance on the main chain and computes additional parameters that are used in later steps of the setup.
+
+### Establishing partner chain on Cardano
+
+Each partner chain is identified by the Genesis UTXO and it has some governance.
+Use `<node-executable> smart-contracts governance init -c <GENESIS_UTXO> -g <GOVERNANCE_AUTHORITY> -t <THRESHOLD> -k <PAYMENT_KEY_FILE>`
+command to spend the UTXO that will become the Genesis UTXO of your chain.
+The Genesis UTXO has to be spendable by the payment key.
+This command initializes governance of the chain as M of N multisig parametrized by GOVERNANCE_AUTHORITY and THRESHOLD parameters.
```

Comment:
While it is implicitly clear what `GENESIS_UTXO` is, it isn't explicit what `GOVERNANCE_AUTHORITY` is and where it comes from. Same goes for `THRESHOLD`: not clear with what value/type that would be invoked.

Maybe it would be nice to have example invocations for all commands in here? And the explicit listing of the argument names etc can be left to https://github.com/input-output-hk/partner-chains/pull/811 once that is merged?